.github/workflows/kind_integration.yml

```diff
@@ -50,6 +50,9 @@ jobs:
           docker save "$DOCKER_REGISTRY/$image:$TAG" > $ARCHIVES/$image.tar || tee save_fail &
         done
 
+        # save cli's as artifacts
+        cp -r ./target/cli $ARCHIVES/cli
```

Comment:
We end up with two copies of the cli binaries uploaded as artifacts, the other copy being inside the docker images. But we do this so we don't have to instantiate a docker image in order to retrieve them below in the windows test (like we do in the kind tests below), right?

Reply:
Yeah, Loading linux build docker images don't seem to work on windows. They err out mentioning that its a different ref format.